api/plan.go

```diff
@@ -94,6 +94,11 @@ const (
 	//
 	// Currently, we only support GitLab EE/CE OAuth login.
 	Feature3rdPartyLogin FeatureType = "bb.feature.3rd-party-login"
+
+	// FeatureSyncProjectMemberFromVCS allows user to sync project member from a configured VCS
+	//
+	// Currently, we only support GitLab EE/CE for this feature.
+	FeatureSyncProjectMemberFromVCS FeatureType = "bb.feature.sync-project-member-from-vcs"
```

Comment:
I'm thinking whether we should just reuse Feature3rdPartyLogin, from product perspective. Or maybe call it Feature3rdPartyAuth (authentication+authorization).

@tianzhou wdyt?

Reply:
Feature3rdPartyAuth SGTM